lerobot/common/robot_devices/control_utils.py

```diff
@@ -120,14 +120,16 @@ def predict_action(observation, policy, device, use_amp):
     return action
 
 
-def init_keyboard_listener():
+def init_keyboard_listener(assign_rewards=False):
     # Allow to exit early while recording an episode or resetting the environment,
     # by tapping the right arrow key '->'. This might require a sudo permission
     # to allow your terminal to monitor keyboard events.
```

Comment:
I made this comment on #518 but I'll add it it here since this PR contains all the additions of the previous one. 
Should we add some comments here to explain `assign_rewards`?
```suggestion
    """
    Initializes a keyboard listener to enable early termination of an episode 
    or environment reset by pressing the right arrow key ('->'). This may require 
    sudo permissions to allow the terminal to monitor keyboard events.

    Args:
        assign_rewards (bool): If True, allows annotating the collected trajectory 
        with a binary reward at the end of the episode to indicate success.
    """
```

Reply:
ah yes that's nice